.github/workflows/regular.yml

```diff
@@ -80,6 +80,13 @@ jobs:
       - name: Build
         run: just build --sizes
 
+      - name: Run integration tests
+        run: just test-integration
```

Comment:
instead of these 2 commands, you should probably run `test-post-voting`
it also includes staking router checks

Reply:
done